tests/Functional/SearchTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Tests\Functional;
+
+use Setono\SyliusMeilisearchPlugin\Engine\SearchEngine;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/** @group functional */
+final class SearchTest extends WebTestCase
+{
+    private static KernelBrowser $client;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        self::ensureKernelShutdown();
+        self::$client = static::createClient(['environment' => 'test', 'debug' => true]);
+    }
+
+    public function testItProvidesSearchResults(): void
+    {
+        /** @var SearchEngine $searchEngine */
+        $searchEngine = self::getContainer()->get(SearchEngine::class);
+        $result = $searchEngine->execute('jeans');
+
+        self::assertSame(8, $result->getHitsCount());
+    }
+
+    public function testItSortsSearchResultsByLowestPrice(): void
+    {
+        /** @var SearchEngine $searchEngine */
+        $searchEngine = self::getContainer()->get(SearchEngine::class);
+        $result = $searchEngine->execute('jeans', ['sort' => 'price:asc']);
+
+        self::assertSame(8, $result->getHitsCount());
+
+        $previousKey = null;
+        foreach ($result->getHits() as $key => $hit) {
+            if ($previousKey === null) {
+                $previousKey = $key;
+
+                continue;
+            }
+
+            $previousHit = (array) $result->getHit($previousKey);
+            self::assertGreaterThanOrEqual($previousHit['price'], $hit['price']);
+            $previousKey = $key;
+        }
+    }
+
+    public function testItSortsSearchResultsByNewestDate(): void
+    {
+        /** @var SearchEngine $searchEngine */
+        $searchEngine = self::getContainer()->get(SearchEngine::class);
+        $result = $searchEngine->execute('jeans', ['sort' => 'createdAt:desc']);
```

Comment:
This small thing `createdAt:desc` made me think about whether we should snake case properties inside Meilisearch? If there would ever be a problem with camel case? I am thinking about web servers ignoring case, browsers ignoring case or whatever. I don't know. Do any of you know? It will give a lot of other problems though :)

Reply:
IMO it shouldn't make any troubles.